documentation/asciidoc/topics/proc_uninstalling.adoc

```diff
@@ -0,0 +1,31 @@
+[id='uninstalling-helm-chart_{context}']
+= Uninstalling the Helm chart
+
+[role="_abstract"]
+Uninstall the {brandname} chart to remove pods and other deployment artifacts.
+
+[NOTE]
+====
+This procedure shows you how to uninstall the {brandname} chart and generated with the `helm` and `oc` clients.
```

Comment:
I think you missed a word :slightly_smiling_face:. "generated authentication secret" or just "generated secret"?

Reply:
I blame jetlag for this. fixed now. thanks!